recipes/echopype/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.21" %}
+
+package:
+  name: echopype
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/e/echopype/echopype-{{ version }}.tar.gz
+  sha256: d711d4ac6c94e84e1488e01bb80abaca412f6bf58d4b86d7f46bf4985d29a728
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - click
+    - numpy
+    - pandas
+    - scipy
+    - matplotlib
+    - netCDF4
+    - xarray
+    - pytz
+
+test:
+  imports:
+    - echopype
+
+about:
+  home: https://github.com/OSOceanAcoustics/echopype
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Enhancing interoperability and scalability in analyzing water column sonar data.
+  doc_url: https://echopype.readthedocs.io/en/latest/
+  dev_url: https://github.com/OSOceanAcoustics/echopype
+
+extra:
+  recipe-maintainers:
+    - lsetiawan
+    - leewujung
```

Comment:
@leewujung are you OK with being a maintainer?

Reply:
Yes sounds good. Thanks!